nodejs/private/ts_project_options_validator.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
```

Comment:
How did this transpiled file get here? Source of truth is still `packages/typescript/internal/ts_project_options_validator.ts`?

Reply:
correct, and there's a `diff_test` in the BUILD file next to that `.ts` file.

The reasoning is: we no longer have the `#BEGIN/END-INTERNAL` fencing trick because `rules_nodejs` promises to be equally usable from sources vs. some distro artifact. We don't want to have `load` statements here in `rules_nodejs` that try to bring in typescript compilation, because `rules_nodejs` by itself can't do it.

Reply:
I see ```diff_test(
    name = "check_validator.js",
    # We are copying across packages here, so don't use generated_file_test, just print a command
    failure_message = "\nRun:\n    cp dist/bin/packages/typescript/internal/*.js nodejs/private",
    file1 = "compile",
    file2 = "//nodejs/private:ts_project_options_validator.js",
    tags = ["no-bazelci-windows"],
)```